cppguide.html

```diff
@@ -4725,14 +4725,15 @@ <h4 class="stylepoint_subsection">Class Data Members</h4>
 <h4 class="stylepoint_subsection">Struct Data Members</h4>
 
 <p>Data members of structs, both static and non-static,
-are named like ordinary nonmember variables. They do not have
-the trailing underscores that data members in classes have.</p>
-
-<pre>struct UrlTableProperties {
-  string name;
-  <del>int num_entries;</del>
-  <strong>int numEntries;</strong>
-  static Pool&lt;UrlTableProperties&gt;* pool;
+are named like ordinary nonmember variables, <strong>but with a
+trailing underscore.</strong> <del>They do not have
+the trailing underscores that data members in classes have.</del></p>
+
+<pre>struct TableInfo {
+  <del>string table_name_;  // OK - underscore at end.</del>
```

Comment:
The idea was to clearly mark changes to the original google style guide. So do not edit without <del> and <strong> e.g. TableInfo change. and use the orignal variable names. Thanks for adapting though, should have told you.

Reply:
Ok sorry, I copy/pasted the example for classes :)

Reply:
Should be fine now. I am re-using the existing example, deleting/stronging the members.

Reply:
Thanks!